recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure -fPIC is not in CXXFLAGS (that some conda packages may
+# add), otherwise omniscidb server will crash when executing generated
+# machine code:
+export CXXFLAGS="`echo $CXXFLAGS | sed 's/-fPIC//'`"
+
+# Fixes https://github.com/Quansight/pearu-sandbox/issues/7
+#       https://github.com/omnisci/omniscidb/issues/374
+export CXXFLAGS="$CXXFLAGS -Dsecure_getenv=getenv"
+
+# Fixes `error: expected ')' before 'PRIxPTR'`
+export CXXFLAGS="$CXXFLAGS -D__STDC_FORMAT_MACROS"
+
+# resolves `fatal error: boost/regex.hpp: No such file or directory`
+echo -e "#!/bin/sh\n$(which nvcc || omniscidb_build_sh_could_not_find_nvcc) -I$PREFIX/include \$@" > $PWD/nvcc
+chmod +x $PWD/nvcc
+export PATH=$PWD:$PATH
+
+# Remove --as-needed to resolve undefined reference to `__vdso_clock_gettime@GLIBC_PRIVATE'
+export LDFLAGS="`echo $LDFLAGS | sed 's/-Wl,--as-needed//'`"
+
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}"
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCUDA_TOOLKIT_ROOT_DIR=$CUDA_HOME"
+
+# Required for building on a CUDA enabled system without libcuda.so.1
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_LIBRARY_PATH=$CUDA_HOME/lib64/stubs"
+
+mkdir -p build
+cd build
+
+# TODO: when building on a system with no GPUs, using
+# -DENABLE_TESTS=off will save build time
+
+cmake -Wno-dev \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX/opt/omnisci \
```

Comment:
`$PREFIX/opt` is strange. There is no `/usr/opt`.

Reply:
There is actually `$PREFIX/opt` already, for instance, `maven` uses it. Btw, `/opt` is a part of the standard filesystem hierarchy.

Other alternative to `$PREFIX/opt/omnisci` is:
```
$PREFIX/omnisci
```
which would be less standard, although, many packages use it (e.g. `go`, `jre`, `ssl`, etc).

I didn't use `$PREFIX/share/omnisci` because `/usr/share` is assumed to contain "Architecture-independent (shared) data." The content of `omnisci/` would be too far from that.

Out of all the available options, `$PREFIX/opt/omnisci` seemed most appropriate to me for the software package like omniscidb is.